samples/dlp_inspect_string.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(
+  projectId = 'YOUR_PROJECT_ID',
+  stringToInspect = 'My name is Gary and my email is gary@example.com'
+) {
+  // [START dlp_inspect_string]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'YOUR_PROJECT_ID';
+  // const stringToInspect = 'My name is Gary and my email is gary@example.com';
+
+  // Imports the Google Cloud Data Loss Prevention library
+  const DLP = require('@google-cloud/dlp');
+
+  // Instantiates a client
+  const dlp = new DLP.DlpServiceClient();
+
+  async function inspectString() {
+    // Construct request
+    const request = {
+      parent: dlp.projectPath(projectId),
+      item: {
+        value: stringToInspect,
+      },
+      inspectConfig: {
+        // The infoTypes of information to match
+        infoTypes: [
+          {name: 'PHONE_NUMBER'},
+          {name: 'EMAIL_ADDRESS'},
+          {name: 'CREDIT_CARD_NUMBER'},
+        ],
+        // Whether to include the matching string
+        includeQuote: true,
+      },
+    };
+
+    // Run request
+    const [inspectResponse] = await dlp.inspectContent(request);
+    const findings = inspectResponse.result.findings;
+    console.log(`Findings: ${!findings.length ? 'None' : ''}`);
+    findings.forEach(finding => {
```

Comment:
Personally I think `for` loops are far more readable.  Can we do that instead?

Reply:
Done